terraform/dev.tfvars

```diff
@@ -1,8 +1,6 @@
 aws_account = "dev"
-system_dns_zone_id = "Z1QGLFML8EG6G7"
-system_dns_zone_name = "dev.cloudpipeline.digital"
+env_dns_zone_id = "Z1QGLFML8EG6G7"
```

Comment:
I'm not sure I understand why this variable was renamed? I'm not sure that "env" in the new variable name is very clear. When reading through this, I initially assumed that "env" referred to environment variables.

Is there a reason to move away from `system_dns_zone*` and `apps_dns_zone*` as names for the system components and deployed apps respectively?